docs/tutorials/multimodal/object_detection/detection_high_performance_finetune_coco.md

```diff
@@ -0,0 +1,186 @@
+You can also specify the `save_path` like below while creating the MultiModalPredictor.
+For more information about save and load the model,
+see :ref:`sec_automm_detection_save_and_load`. 
+
+```
+predictor = MultiModalPredictor(
+    save_path="./this_is_a_save_path",
+    ...
+)
+```
+
+Print out the time and we can see that it takes almost 5 hours.
+
+```python
+print("This finetuning takes %.2f seconds." % (end - start))
+```
+
+```
+This finetuning takes 17779.09 seconds.
+```
+
+It does take a lot of time but let's look at its performance.
+To evaluate the model we just trained, run:
+
+```python
+predictor.evaluate(test_path)
+```
+
+And the evaluation results are shown in command line output. The first value `0.740` is mAP in COCO standard, and the second one `0.932` is mAP in VOC standard (or mAP50). For more details about these metrics, see [COCO's evaluation guideline](https://cocodataset.org/#detection-eval).
+
+```
+ Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.740                                                                               
+ Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.932                                                                               
+ Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.819                                                                               
+ Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.483                                                                               
+ Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.617                                                                               
+ Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.792                                                                               
+ Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.569                                                                               
+ Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.811                                                                               
+ Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.827                                                                               
+ Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.603                                                                               
+ Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.754                                                                               
+ Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.866  
+```
+
+Under this high performance finetune setting, it took 5 hours but reached `mAP50 = 0.932` on VOC!
```

Comment:
We can write one sentence here to compare with performance using yolov3_mobilenetv2. Let users have a clear understanding what is the accuracy and speed tradeoff is.

Reply:
Done, also added the comparison in yolov3 finetune tutorial